src/OrleansAzureUtils/Hosting/AzureConfigUtils.cs

```diff
@@ -125,7 +127,8 @@ private static IEnumerable<DirectoryInfo> FindAppDirectoryLocations()
                 if (appRootPath != null)
                     locations.Add(new DirectoryInfo(appRootPath));
             });
-
+#if !NETSTANDARD_TODO
+            //System.Web namespace is deprecated in netstandard, need to find a alternative way to get {ServerRoot}
```

Comment:
Yeah I'm not sure how useful it is, the comment on it suggested that it is used for finding server root in web roles running in IIS web apps or IIS express.. 

More than happy to remove it if this block of code is obsolete